### DISCUSSION TITLE
ConcurrentObservableRangeCollection

### DISCUSSION THREAD

**Discussion by MaxFmi** [CommunityToolkit/Maui#491] [New Feature Discussions]
We have a contract for ListViewItems. This contract gets implemented from multiple modules. To avoid thread issues we're using our own thread safe implementation of ObservableRangeCollection. It would be great to have a thread safe variant in the .NET MAUI CommunityToolkit. Maybe ConcurrentObservableRangeCollection

**Comment by bijington** [comment]
This looks like it would be a better consideration for the .NET Toolkit over at: https://github.com/CommunityToolkit/dotnet

I can't see how to transfer a discussion there, is it possible to open a new discussion there please?

**Reply by MaxFmi** [reply]
Opened a new discussion under https://github.com/CommunityToolkit/dotnet -> https://github.com/CommunityToolkit/dotnet/discussions/346

**Comment by CliffAgius** [comment]
Closed as answered
